README.md

```diff
@@ -39,7 +40,7 @@ It significantly reduces traditional CDN traffic and cost while delivering media
 - Supports live and VOD streams over HLS or MPEG-DASH protocols
 - Supports multiple HTML5 video players and engines:
   - Engines: Hls.js, Shaka Player
-  - Video players: JWPlayer, Clappr, Flowplayer, MediaElement, VideoJS, Plyr, DPlayer, Player.js and others
+  - Video players: [Vidstack](https://www.vidstack.io/), [Clappr](http://clappr.io/), [MediaElement](https://www.mediaelementjs.com/), [Plyr](https://plyr.io/), [DPlayer](https://dplayer.diygod.dev/), [OpenPlayerJS](https://www.openplayerjs.com/) and others
```

Comment:
others that support Hls.js or Shaka video engines + via custom integration with the library API